cluster-autoscaler/core/static_autoscaler.go

```diff
@@ -285,7 +285,7 @@ func (a *StaticAutoscaler) RunOnce(currentTime time.Time) errors.AutoscalerError
 			scaleDownStart := time.Now()
 			metrics.UpdateLastTime("scaleDown", scaleDownStart)
 			result, typedErr := scaleDown.TryToScaleDown(allNodes, allScheduled, pdbs)
-			metrics.UpdateDuration("scaleDown", scaleDownStart)
```

Comment:
Maybe just leave it for sanity checking? But no strong feelings, I can remove it.